### DISCUSSION TITLE
anyone has experiences with running Kibot on Gitea?

### DISCUSSION THREAD

**Discussion by nopeppermint** [INTI-CMNB/KiBot#457] [Other platforms]
anyone has experiences with running Kibot on Gitea?
Gitea seems to have now a runner similar to Github Actions https://docs.gitea.com/next/usage/actions/comparison

**Comment by set-soft** [comment]
Hi @nopeppermint !
From the text you point it looks like this is fairly compatible with GitHub. I think the best is to just give it a try and post any errors here so we can investigate the reason.

**Comment by nopeppermint** [comment]
FYI it seems Kibot is running under Woodpecker CI (which is based on Drone CI ) on codeberg.org.

codeberg.org is based on forgejo.org which is a "fork" of gitea.

https://codeberg.org/ccoenen/Codeberg-KiCad-KiBot-Example

**Reply by fanningert** [reply]
I can confirm, it is working with Gitea Actions. The are compatible with Github Actions. I currently run gitea and the runner (Act Runner) for Gitea Actions as docker in a LXC container and everything is working fine.
